File: CC/Codigos_Fonte_Relatorio_1/micro11/micro11.java

public class micro11{

    public static int verifica(int n){
	int res;
        if(n > 0)
		res = 1;
        else if (n < 0)
		res = -1;
	else
		res = 0;
	return res;
    }

    public static void main(String[] args){
        int numero, x;

        System.out.println("Digite um numero: ");
        numero = Integer.parseInt(System.console().readLine());

	if (x == 1)
	        System.out.println("Número positivo");
	else if (x == 0)
	        System.out.println("Zero");
	else
	        System.out.println("Número negativo");
    }

}

File: VersaoFinal/Latex/codigos/micro11.lua
function verifica(n)
  local res
  
  if n > 0 then
    res = 1
  elseif n < 0 then
    res = -1
  else 
    res = 0
  end
  
  return res
end

local numero, x

io.write("Digite um número: ")
numero = tonumber(io.read())

x = verifica(numero)

if x == 1 then
  io.write("Numero positivo\n")
  
elseif x == 0 then 
  io.write("Zero\n")
  
else
  io.write("Numero Negativo\n")
  
end

